### PR TITLE
(CLI) Update cli list functions

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

`yarn cli reporter list`
`yarn cli listener list`
`yarn cli delegator reporterList`

now will be printing out datafeed pair name if possible
image

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: cli/src/delegator.ts [hunk 1]
```diff
@@ -296,6 +298,20 @@ export function reporterListHandler() {
...
+      for (const reporter of result) {
+        if (!reporter.contract) {
+          continue
+        }
+        const url = new URL(AGGREGATOR_ENDPOINT)
+        url.searchParams.append('address', reporter.contract[0])
+        const aggregatorResult = (await axios.get(url.toString())).data
+        console.log(aggregatorResult)
+        if (aggregatorResult && aggregatorResult.name) {
+          reporter.name = aggregatorResult.name
```
Comment (martinkersner) [inline cli/src/delegator.ts:311]: If I understand it correctly, you are in-place modifying data inside of `result` which are later printed on line 315. We should avoid this side effect problem solving. Is there a way to do it without it?

File: cli/src/delegator.ts [hunk 2]
```diff
@@ -296,6 +298,20 @@ export function reporterListHandler() {
...
       const result = (await axios.get(endpoint)).data
+
+      for (const reporter of result) {
+        if (!reporter.contract) {
+          continue
+        }
+        const url = new URL(AGGREGATOR_ENDPOINT)
+        url.searchParams.append('address', reporter.contract[0])
+        const aggregatorResult = (await axios.get(url.toString())).data
+        console.log(aggregatorResult)
```
Comment (martinkersner) [inline cli/src/delegator.ts:309]: I guess this used to used to debugging 🤔 Let's remove it.

**Review by martinkersner — CHANGES_REQUESTED** [review]

File: cli/src/delegator.ts (hunk 1 above)
Comment (nick-bisonai) [reply cli/src/delegator.ts:311]: rather than adding to result data, each elements can be printed within iteration, how do you think?

Comment (martinkersner) [reply cli/src/delegator.ts:310]: In that case, I think it is better to keep it as it is.

**Review by martinkersner — APPROVED** [review]
LGTM!

**Pull request merged**